### PR TITLE
[1LP][RFR] Automating copy catalog item with tags

### PR DESCRIPTION
**Pull request by niyazRedhat**
### PRT Run
{{pytest: cfme/tests/services/test_catalog_item.py::test_copy_catalog_item_with_tags }}

No PRT run, the BZ is still in Post state. I have tested locally with ipdb.

### CODE REVIEW

File: cfme/tests/services/test_catalog_item.py [hunk 1]
```diff
@@ -584,10 +584,10 @@ def test_catalog_item_price_currency():
 
 
 @pytest.mark.meta(coverage=[1740399])
-@pytest.mark.manual
 @pytest.mark.ignore_stream('5.10')
+@pytest.mark.meta(blockers=[BZ(1740399, forced_streams=['5.11'])])
```
Comment (sshveta) [inline cfme/tests/services/test_catalog_item.py]: Also add automates please 
    
    @pytest.mark.meta(blockers=[BZ(1740399, forced_streams=["5.11"])], automates=[1740399])

**Comment by niyazRedhat** [comment]
The BZ is post State for the last 3 month I think we shouldn't wait for BZ to be verified.

File: cfme/tests/services/test_catalog_item.py [hunk 2]
```diff
@@ -608,4 +607,12 @@ def test_copy_catalog_item_with_tags():
             3.
             4. Tags to be copied with catalog item
     """
-    pass
+    generic_catalog_item.add_tag(tag)
+    new_cat_item = generic_catalog_item.copy()
+    request.addfinalizer(new_cat_item.delete_if_exists)
+
+    assert any([
```
Comment (john-dupuy) [inline cfme/tests/services/test_catalog_item.py]: Q: shouldn't this be `all` rather than `any`?

**Comment by sshveta** [comment]
The test is skipped as it marked with a BZ. 
Some other unrelated test failed from the file.
@niyazRedhat : You can run individual test changed by providing ' -k' in pytest run command

**Pull request merged**